app/controllers/api/v1/characters_controller.rb

```diff
@@ -112,6 +120,12 @@ def find_post
     access_denied unless @post.visible_to?(current_user)
   end
 
+  def find_template
+    return unless params[:template_id].present?
+    return if params[:template_id] == '0' # used to filter for templateless characters
```

Comment:
Shouldn't this do... something...?

Reply:
Up in #index it checks if `params[:template_id].present?` (which it would be) and then filters on `template_id: @template&.id` which means it will filter on `template_id: nil` since we quit out of this method without setting `@template`

Reply:
Looks good then!

Reply:
(Might not be the _clearest_ code, but I assume making it clearer would also make it much more convoluted, so.)

Reply:
I can give the comment a few more words without much drama I think :D